docs/repo/dependencies-node-roadmap.md

```diff
@@ -66,11 +66,21 @@ The [`DependenciesTreeViewProvider`][DependenciesTreeViewProvider] traverses dow
 
 > Aside: The `IProjectTree` nodes are intentionally updated from top to bottom as it prevents the Solution Explorer from collapsing expanded nodes during the update. At the very least this would be visually distracting to the user.
 
-[`IDependency`][IDependency]s are not translated directly into `IProjectTree`s. They are first converted to [`IDependencyViewModel`][IDependencyViewModel]s and those in turn become the `IProjectTree`s. This makes it a little easier to create the `IProjectTree`s for targets and groups (e.g. the Assemblies, NuGet, Projects, etc. nodes) which are not themselves [`IDependency`][IDependency]s.
+[`IDependency`][IDependency]s are not translated directly into `IProjectTree`s. They are first converted to [`IDependencyViewModel`][IDependencyViewModel]s and those in turn become the `IProjectTree`s. This makes it a little easier to create the `IProjectTree`s for targets and groups (e.g. the Assemblies, NuGet, Projects, etc. nodes) which are not themselves [`IDependency`][IDependency]s. In some cases a [`IDependencyModel`][IDependencyModel] may be converted directly to a [`IDependencyViewModel`][IDependencyViewModel].
 
 ### Identifiers
 
-> TODO: Describe the role and implementation of the [`IDependency`][IDependency]`.Id` property.
+#### `IDependencyModel` Identifiers
```

Comment:
There are 3 `IDependencyModel`'s you can add links to

Reply:
👍 Updated.